test/unit_tests/api/test_processes_ws.py

```diff
@@ -139,10 +139,12 @@ def test_websocket_process_detail_workflow(test_client, long_running_workflow):
         with test_client.websocket_connect("api/processes/all/?token=") as websocket:
             # Check the websocket messages.
             # the initial process details.
-            data = websocket.receive_text()
-            process = json_loads(data)["process"]
-            assert process["workflow_name"] == "long_running_workflow_py"
-            assert process["status"] == ProcessStatus.RUNNING
+
+            # WITH AN UPDATE IT DOES NOT SEND THIS ANYMORE
+            # data = websocket.receive_text()
+            # process = json_loads(data)["process"]
+            # assert process["workflow_name"] == "long_running_workflow_py"
+            # assert process["status"] == ProcessStatus.RUNNING
```

Comment:
Worth looking into later in a separate ticket?

Reply:
Created an issue for this: #309